### PR TITLE
Add deployed date to files

### LINKED ISSUE #5: Add deployed date and time to variables and json file

### PR DESCRIPTION
**Pull request by edthedev**
- Remove attempt to deploy the template
- Add deploy date and time to JSON file code variable.

Closes #5

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ jobs:
         make build
     - name: Post to Splunk SOAR API
       run: |
-        make deploy
+        # make deploy
+        echo "WARNING: Uncomment the line above to start deploying."
```
Comment (ddriddle) [inline .github/workflows/deploy.yml:17]: I would like to see this change reverted.

**Comment by edthedev** [comment]
Per discussion - we would rather deploy an empty test app than have the template have deployment disabled.

## Tasks

- [ ] re-enable deployment (uncomment makefile)
- [ ] add a trivial test_connectivity that just reaches out to a known public IP address
- [ ] update app.json with clear labels that the app is a test app.

File: Makefile [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 SRCS_DIR:=src/ph$(MODULE)
 TSCS_DIR:=tests
 SOAR_SRCS:=$(shell find $(SRCS_DIR) -type f)
 SRCS:=$(shell find $(SRCS_DIR) -name '*.py')
 TSCS:=$(shell find $(TSCS_DIR) -name '*.py')
-TAG_FILES:=$(addprefix $(SRCS_DIR)/, $(PACKAGE).json app.py)
-HASH_FILES:=$(addprefix $(SRCS_DIR)/, app.py)
+VERSION_FILES:=$(addprefix $(SRCS_DIR)/, $(PACKAGE).json app.py)
+GITHUB_DEPLOYED:=$(shell date -u +%FT%X.%6NZ)
```
Comment (ddriddle) [inline Makefile:11]: Would BUILD_TIME be a better name?

**Comment by mpitcel** [comment]
Test Comment

File: Makefile [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+PACKAGE:=app
 
 SRCS_DIR:=src/ph$(MODULE)
 TSCS_DIR:=tests
 SOAR_SRCS:=$(shell find $(SRCS_DIR) -type f)
 SRCS:=$(shell find $(SRCS_DIR) -name '*.py')
 TSCS:=$(shell find $(TSCS_DIR) -name '*.py')
-TAG_FILES:=$(addprefix $(SRCS_DIR)/, $(PACKAGE).json app.py)
-HASH_FILES:=$(addprefix $(SRCS_DIR)/, app.py)
+VERSION_FILES:=$(addprefix $(SRCS_DIR)/, $(PACKAGE).json app.py)
```
Comment (ddriddle) [inline Makefile:10]: I think VERSIONED_FILES may be more clear.

File: src/phillinois_app/app.py [hunk 4]
```diff
@@ -19,6 +19,7 @@
 
 __version__ = 'GITHUB_TAG'
 __git_hash__ = 'GITHUB_SHA'
+__deployed__ = 'GITHUB_DEPLOYED'
```
Comment (ddriddle) [inline src/phillinois_app/app.py:22]: I think this should also be `__build_time__` or `__built__`

**Review by ddriddle — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (edthedev) [reply .github/workflows/deploy.yml:17]: Let's take care of this in a dedicated change at #8

File: Makefile (hunk 2 above)
Comment (edthedev) [reply Makefile:11]: These are the names we used in the other projects. I'm down with updating the names here, this is where to get the names right so they move to other projects.

File: src/phillinois_app/app.py (hunk 4 above)
Comment (edthedev) [reply src/phillinois_app/app.py:22]: Done.

File: Makefile (hunk 2 above)
Comment (edthedev) [reply Makefile]: Done.

**Review by mpitcel — APPROVED** [review]

**Review by ddriddle — APPROVED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (edthedev) [reply .github/workflows/deploy.yml]: Update: We were going to take care of this now, but need to work with Engineering to add it to the repo secrets.

**Pull request merged**